.circleci/config.yml

```diff
@@ -63,7 +68,6 @@ commands:
               echo "this is not RC version"
             fi
             echo "export KLAYTN_VERSION=$(go run build/rpm/main.go version)" >> $BASH_ENV
-            
```

Comment:
Please remove unnecessary line removal.

Reply:
which line?

Reply:
This empty line(=line66) has been removed

Reply:
Right. It removed now :)

Reply:
Oh, it is intended. I understood.